Makefile

```diff
@@ -22,6 +22,10 @@ cover-pkg:
 	go test -coverprofile cover.out $(PKG)
 	@grep -v mode: cover.out >> cover-all.out
 
+.PHONY: deps
+deps:
+	dep ensure --vendor-only
```

Comment:
To work around 1.7 issue we may need to create a travis matrix (like this https://github.com/jaegertracing/jaeger-client-go/blob/master/.travis.yml#L8) and set an env var for 1.7 like MANUAL_DEPS=true so that we only run `dep ensure` when it's not set